src/modules/menu/components/advancedTools/ImportFile/ImportFile.vue

```diff
@@ -0,0 +1,71 @@
+<script setup>
+import { ref } from 'vue'
+
+import ImportFileOnlineTab from '@/modules/menu/components/advancedTools/ImportFile/ImportFileOnlineTab.vue'
+import ImportFileLocalTab from '@/modules/menu/components/advancedTools/ImportFile/ImportFileLocalTab.vue'
+
+const selectedTab = ref('online')
+</script>
+
+<template>
+    <div data-cy="import-file-content">
+        <ul class="nav nav-tabs" role="tablist">
+            <li class="nav-item" role="presentation">
+                <button
+                    class="nav-link py-1"
+                    :class="{
+                        active: selectedTab === 'online',
+                    }"
+                    data-bs-toggle="tab"
+                    data-bs-target="#nav-online"
```

Comment:
these `data-bs-...` aren't really useful in your case, as you are not relying on the JS from Bootstrap to create the necessary logic to show or hide the tabs, you are doing it yourself by adding or removing the `show` class.
So this `data-bs-xxx` can be safely removed (and the ID on the child too)